src/syrupy/__init__.py

```diff
@@ -1,12 +1,13 @@
 import pytest
+from typing import Any, Optional, List
 
 from .assertion import SnapshotAssertion
 from .serializer import SnapshotSerializer
 from .location import TestLocation
 from .session import SnapshotSession
 
 
-def pytest_addoption(parser):
+def pytest_addoption(parser: Any) -> None:
```

Comment:
These pytest hooks can likely be updated to use proper types (can do that in a follow up).